.travis.yml

```diff
@@ -25,11 +25,9 @@ services:
 matrix:
   include:
   - python: "2.7"
-    # When set to 1, tests/run_travis.sh will only compile the sources using python -m compileall
-    env: SHOTGUN_COMPILE_ONLY=0
-  - python: "3.6"
-    # We do not support Python 3 for the moment, but we will be compilable at the very least.
-    env: SHOTGUN_COMPILE_ONLY=1
```

Comment:
Maybe we could also test with Qt5 for 2.7 if that's relatively easy to add?

Reply:
Added.  Also we're now running integration tests on Python 3 on CI, and I've fixed PySide2 issues so we're no longer skipping the PySide2 tests.